interface/fax/fax_dispatch.php

```diff
@@ -463,29 +463,9 @@ function mergeTiffs()
 <html>
 <head>
 
-    <?php Header::setupHeader(['opener', 'datetime-picker', 'jquery-ui']);?>
+    <?php Header::setupHeader(['opener', 'datetime-picker']);?>
     <title><?php echo xlt('Dispatch Received Document'); ?></title>
 
-<style>
-
-td, input, select, textarea {
- font-size: 10pt;
-}
-
-.itemtitle {
- font-weight: bold;
-}
-
-div.section {
- border: solid;
- border-width: 1px;
- border-color: #0000ff;
- margin-left: 2em;
- padding: 1em;
-}
-
-</style>
-
 <script language="JavaScript">
 
     <?php require($GLOBALS['srcdir'] . "/restoreSession.php"); ?>
```

Comment:
Hello @bradymiller and @tywrenn 

I found in our codebase have a lot of  `<script language="JavaScript">` , `<style type="text/css">` and `<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"	"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">` code

Are they can replace to `<script>`, `<style>` and `<!DOCTYPE html>` ?

Reply:
@stu01509 Just added this to the styling guide since I actually have been removing these sort of things ever since I started. The language, type, and added XHTML tags to the doctype are all not HTML5 compliant and must be removed.

Reply:
OK, I will create a PR do to this.

Reply:
> The language, type, and added XHTML tags to the doctype are all not HTML5

Please, please be careful on this and ensure the scripts are really compliant and not arbitrarily change this. There are issues and several sins involved here. Just saying....

Reply:
@sjpadgett XHTML is a totally different language than HTML5 and has less attributes than it does (which is a big issue if you want W3C compliance).
As far as the type attributes and stuff like that go, I knew they weren't compliant from the start. Example of W3C validator giving a warning about it:
![image](https://user-images.githubusercontent.com/46497068/81871167-44eebc80-9545-11ea-9f49-f91fe1d790c8.png)

Reply:
> As far as code breakage from this

umm wasn't my point!


> XHTML is a totally different language than HTML5

Is however. There will/can be some hard to very subtle problems that can arise from changing doctypes. I'm pointing out that when changing this, to test fully and carefully. Not just that the UI comes up and leave at that. You guys are just going to need to trust me on this!